### ISSUE TITLE
[Bug Report][3.1.15] Autocomplete tab key is not working on Firefox

### ISSUE THREAD

**Issue by paul-thebaud** [vuetifyjs/vuetify#17214]
### Environment
**Vuetify Version:** 3.1.15
**Vue Version:** 3.2.47
**Browsers:** Mozilla Firefox
**OS:** Linux x86_64

### Steps to reproduce
- Focus the second autocomplete field using keyboard (tab)
- Select an item using keyboard arrows and press enter to validate
- Press tab key

### Expected Behavior
The button after the two fields should be focused on the last tab keypress (since it is the next following element).

### Actual Behavior
Focus is reset to the document instead of being passed to the button.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNqlUk1PhDAQ/StNzwsViRdczXo08R8se+BjjFVamnbAmM3+d6cFWQRNNnqCeW86781r90f+YEzcd8AzvkVQpikQ7nPN2LaPCmPCbyhUIfVYhbpqNRIEdgKHMx22VUtzAOFMMNYUJTR3OX/yX5bkfE7qQgFxoWdBZZJcOSLDd86Jvwhf/y68oC4WLvGcC2M5DkLprEd8b/L1Mj2PTQn7Ysh+K2Z3QqWrrDTIHGD3dTU0ySELJtkd28+c8EcCfdSbFUi7rsF0Ag+3XnoQIxm+4VKZ1iJZNPGrazW9lqNvpXQCQflkLCAeo+fk65y/IBqXCVHVmo7V0MjexhpQaKPEjtqE7TRKBVHdql0ap/FVVAIWcSJq6XDOxuBUVNr23YGlWZPVoIby+eMCxbGThJI4uRkkRiyiJ+C8xmq2IMEebGRB12DBXrrZ4tgP2y06Vht6E6dcnyj+Ruo3t0i+ciH18cb/u3qY5kcdvOJpw+d+ycFwkB8+AShxSXw=)

### Other comments
- This issue is only occuring with Firefox (tested chrome, edge and firefox)
- This has an high impact on accessibility
- I tried to create a PR, but failed. It seems to be related to a [different behavior of Firefox](https://stackoverflow.com/questions/24809835/focusin-and-focusout-methods-not-working-in-firefox) for "focusin" and "focusout" events (if yes, it is probably related to the [VAutocomplete VList event bindings](https://github.com/vuetifyjs/vuetify/blob/next/packages/vuetify/src/components/VAutocomplete/VAutocomplete.tsx#L304-L305).

**Comment by yuwu9145** [comment]
could be caused by #17152
